pageserver/src/storage_sync/index.rs

```diff
@@ -278,8 +278,7 @@ pub struct IndexPart {
 }
 
 impl IndexPart {
-    pub const FILE_NAME: &'static str = "index_part";
-    pub const FILE_EXTENSION: &'static str = "json";
+    pub const FILE_NAME: &'static str = "index_part.json";
 
```

Comment:
I meant to extract this change to a separate PR but forgot. 

We always use FILE_NAME and FILE_EXTENSION together, so I didn't see the point of keeping them separate.